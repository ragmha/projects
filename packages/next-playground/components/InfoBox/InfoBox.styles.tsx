import styled from 'styled-components'

export const StyledInfoBox = styled.div`
  margin-top: 20px;
  margin-bottom: 20px;
  padding-top: 20px;
  padding-bottom: 20px;
  border-top: 1px solid #ececec;
  border-bottom: 1px solid #ececec;
`
