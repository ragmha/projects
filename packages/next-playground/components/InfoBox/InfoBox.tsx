import { FC } from 'react'

import { StyledInfoBox } from './InfoBox.styles'

export const InfoBox: FC = ({ children }) => (
  <StyledInfoBox>{children}</StyledInfoBox>
)

export default InfoBox
