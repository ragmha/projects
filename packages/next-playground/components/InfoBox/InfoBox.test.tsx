import InfoBox from './InfoBox'
import { render } from '@utils/test'

describe('InfoBox', () => {
  it('should match the snapshot', () => {
    const { container } = render(<InfoBox>Info</InfoBox>)
    expect(container).toMatchSnapshot()
  })
})
