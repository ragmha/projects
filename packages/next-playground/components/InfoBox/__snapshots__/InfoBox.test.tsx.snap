// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox should match the snapshot 1`] = `
<div>
  <div
    class="sc-AxjAm gEdKsr"
  >
    Info
  </div>
</div>
`;
